cdap-app-fabric/pom.xml

```diff
@@ -218,11 +218,10 @@
       <artifactId>jul-to-slf4j</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.opentable.components</groupId>
-      <artifactId>otj-pg-embedded</artifactId>
-      <scope>test</scope>
+      <groupId>io.zonky.test</groupId>
+      <artifactId>embedded-postgres</artifactId>
```

Comment:
Should this still be in test scope?

Reply:
The scope is set in the dependencyManagement section in the parent pom. That is to make sure all usages are in test scope unless explicitly overridden.